glossary.yml

```diff
@@ -1057,7 +1057,13 @@
       Seti ya vipengele na taratibu zinazotolewa na maktaba moja ya programu au 
       huduma ya tovuti ambayo programu nyingine inaweza kuwasiliana nayo. 
       API sio msimbo, hifadhidata, au seva: ni mahali pa ufikiaji.
-      
+  en:
```

Comment:
should be "uk"

Reply:
As README at https://github.com/carpentries/glosario says, this "must be a single ISO 639 language code" and for Ukrainian language that is "uk"